contracts/interchain-token-service/src/state.rs

```diff
@@ -29,6 +34,7 @@ pub struct ChainConfig {
 
 const CONFIG: Item<Config> = Item::new("config");
 const ITS_CONTRACTS: Map<&ChainNameRaw, Address> = Map::new("its_contracts");
+const FROZEN_CHAINS: Map<&ChainNameRaw, ()> = Map::new("frozen_chains");
 const CHAIN_CONFIGS: Map<&ChainNameRaw, ChainConfig> = Map::new("chain_configs");
```

Comment:
shall we add freeze status in the chain config instead? It also ensures that we only freeze registered chains, and not any chain name